src/frontend/src/app/app.module.ts

```diff
@@ -25,7 +25,7 @@ import { RegistertermsComponent } from './components/register/registerterms/regi
 import { ErrorInterceptor } from './helpers/error.interceptor';
 import { JwtInterceptor } from './helpers/jwt.interceptor';
 import { SearchTermChangeService } from './services/search-term-change.service';
-import { OwlModule } from 'ngx-owl-carousel';
```

Comment:
This module needs to be included or the home page blows up. :(
Once I added that back in, it rendered perfectly fine!